src/cts/src/TechChar.h

```diff
@@ -190,8 +190,8 @@ class TechChar
 
   void create();
   void compileLut(std::vector<ResultData> lutSols);
-  void write(const std::string& file) const;
-  void writeSol(const std::string& file) const;
+  void printChar() const;
+  void printSol() const;
```

Comment:
Use full words.  Char makes me think of the char type not characterization (which I am just guessing is what Char means, I'm not sure).